src/server.rs

```diff
@@ -261,24 +261,25 @@ impl<E: ServerExt> Server<E> {
         self.disconnections
             .send(Disconnected { id, result })
             .map_err(|_| ())
-            .unwrap();
+            .unwrap_or_default();
     }
 
-    pub fn call(&self, call: E::Call) {
-        self.calls.send(call).map_err(|_| ()).unwrap();
+    pub fn call(&self, call: E::Call) -> Result<(), ()> {
```

Comment:
Cargo Clippy doesn't want to let `Result<(), ()>` through because of https://rust-lang.github.io/rust-clippy/master/index.html#/result_unit_err.